index.js

```diff
@@ -27,4 +29,43 @@ app.get('/', (req, res) => {
   );
 });
 
-app.listen(port, () => console.log(`Express App Listening on port ${port}!`));
+app.post('/invite', async (req, res) => {
+  const MAIL_SERVER_EMAIL = ''; // this is the GMAIL you want to send emails from. "nick@gmail.com"
+  const MAIL_SERVER_PASS = ''; // this is your GMAIL password
+  const SENDER_NAME = ''; // specify the name of the person sending an email, for example: "Nick Wong ⚡️🔋"
+
+  const RECORD_ID = req.body.pledgeInviteID;
+  const pledgeInvite = await getPledgeInviteById(RECORD_ID);
+  const { email: recipientEmail } = pledgeInvite;
+
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port: 587,
+    secure: false, // true for 465, false for other ports
+    auth: {
+      user: MAIL_SERVER_EMAIL,
+      pass: MAIL_SERVER_PASS
+    }
+  });
+
+  const baseUrl = 'https://peoplepower.netlify.com/onboarding';
+  const inviteParameter = `?token=${RECORD_ID}`;
+  const inviteLink = baseUrl + inviteParameter;
+
+  const info = await transporter.sendMail({
+    from: `"${SENDER_NAME}" <${MAIL_SERVER_EMAIL}>`, // sender address
+    to: recipientEmail, // list of receivers
+    subject: 'PP POWER invites you!!', // Subject line
+    text: `Welcome to People Power Solar Cooperative! To join, you can create your account via this link: ${inviteLink}`, // plain text body
```

Comment:
so this "text" and subject stuff could be exported to a file called like `EmailTemplates.js` which has two objects that looks something like this: 

```
export const subjects = {
  invite: () => (`PP POWER invites you!!'}
}

export const emailText = {
  invite: (inviteLink) => `Welcome to People Power Solar Cooperative! To join, you can create your account via this link: ${inviteLink}`
}

```

Reply:
you could a similar HTML object!